### PR TITLE
feat: installer populates vector store

### PR DESCRIPTION
**Pull request by TimPietrusky**
## Motivation

- added a new step into the installer, that explains how to use the prompt and it populates the vector store
- made sure that "unpack" can now also delete the downloaded file after it was extracted
- cleaned up "main"
- added translations for de

### CODE REVIEW

**Comment by TimPietrusky** [comment]
@pixelass please add the translations for every other language that is not `de`.

**Review by pixelass — CHANGES_REQUESTED** [review]
missing translations for all languages. I only see german here.

Also please run `npm run sort-json` to sort the keys in i18n files: https://github.com/blib-la/captain/blob/alpha/package.json#L20

**Comment by TimPietrusky** [comment]
@pixelass all languages were added, conflicts solved and everything else too!

File: src/client/pages/[locale]/installer/03.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+		const unsubscribeInitialized = window.ipc.on(
+			buildKey([ID.INSTALL], { suffix: ":initialized" }),
+			() => {
+				setLoading(false);
+			}
+		);
+
+		const unsubscribeError = window.ipc.on(buildKey([ID.INSTALL], { suffix: ":error" }), () => {
+			setLoading(true);
+		});
```
Comment (pixelass) [inline src/client/pages/[locale]/installer/03.tsx]: why does an error set loading to true?

File: src/client/pages/[locale]/installer/03.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+						<Button
+							endDecorator={
+								loading || starting ? <CircularProgress size="sm" /> : null
+							}
+							onClick={() => {
+								setStarting(true);
+								window.ipc.send(buildKey([ID.APP], { suffix: ":ready" }), true);
+							}}
+						>
+							{(() => {
```
Comment (pixelass) [inline src/client/pages/[locale]/installer/03.tsx]: no self calling functions. please use a `let` assignment before the return

**Review by pixelass — CHANGES_REQUESTED** [review]

File: src/client/pages/[locale]/installer/03.tsx (hunk 1 above)
Comment (TimPietrusky) [reply src/client/pages/[locale]/installer/03.tsx]: This was changed to `setLoading(false)` and I added error handling to show the actual problem.

File: src/client/pages/[locale]/installer/03.tsx (hunk 2 above)
Comment (TimPietrusky) [reply src/client/pages/[locale]/installer/03.tsx]: This was updated!

**Comment by TimPietrusky** [comment]
@pixelass 

* use `let`
* added some basic error handling

**Review by pixelass — APPROVED** [review]

**Pull request merged**